### PR TITLE
docs(protocol): fix LibBlockHeader comment on EIP-1559

### PR DESCRIPTION
**Pull request by dionysuzx**

### CODE REVIEW

**Review by dantaik — DISMISSED** [review]

File: packages/protocol/contracts/libs/LibBlockHeader.sol [hunk 1]
```diff
@@ -45,10 +45,10 @@ library LibBlockHeader {
         uint256 extraCapacity
     ) internal pure returns (bytes[] memory list) {
         if (header.baseFeePerGas == 0) {
-            // non-EIP11559 transaction
+            // non-EIP-1159 transaction
```
Comment (davidtaikocha) [inline packages/protocol/contracts/libs/LibBlockHeader.sol:48]:
```suggestion
            // non-EIP-1559 transaction
```

File: packages/protocol/contracts/libs/LibBlockHeader.sol [hunk 2]
```diff
@@ -45,10 +45,10 @@ library LibBlockHeader {
...
             list = new bytes[](15 + extraCapacity);
         } else {
-            // EIP1159 transaction
+            // EIP-1159 transaction
```
Comment (davidtaikocha) [inline packages/protocol/contracts/libs/LibBlockHeader.sol:51]: ditto

File: packages/protocol/contracts/libs/LibBlockHeader.sol [hunk 3]
```diff
@@ -69,7 +69,7 @@ library LibBlockHeader {
         // as [8]byte when hashing the block.
         list[14] = LibRLPWriter.writeBytes(abi.encodePacked(header.nonce));
         if (header.baseFeePerGas != 0) {
-            // non-EIP11559 transaction
+            // EIP-1159 transaction
```
Comment (davidtaikocha) [inline packages/protocol/contracts/libs/LibBlockHeader.sol:72]: ditto

**Review by dantaik — APPROVED** [review]

**Comment by dantaik** [comment]
Screenshot 2023-03-09 at 19 33 15

@davidtaikocha  could you resolve this?

**Comment by dantaik** [comment]
Ok, lets close this one

**Pull request closed without merging**